lib/services/local/sessiontracker.go

```diff
@@ -116,16 +117,46 @@ func (s *sessionTracker) GetActiveSessionTrackers(ctx context.Context) ([]types.
 		return nil, trace.Wrap(err)
 	}
 
-	sessions := make([]types.SessionTracker, len(result.Items))
-	for i, item := range result.Items {
+	sessions := make([]types.SessionTracker, 0)
+	expired := make([]backend.Item, 0)
```

Comment:
```suggestion
	var sessions []types.SessionTracker
	var expired []backend.Item
```

Reply:
Or maybe `sessions` could be slightly overallocated with `len(result.Items)` regardless?